### DISCUSSION TITLE
[Feature] Set specific album as default startup view

### DISCUSSION THREAD

**Discussion by Fmstrat** [immich-app/immich#10641] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When opening Immich, you are currently defaulted to "everything". This can include screenshots, private'ish folders, downloads, etc. Adding an option in preferences to default to an album would allow users to set their default view to something like `Camera` to see just items in their camera roll.

This would work well combined with https://github.com/immich-app/immich/discussions/10640, as you could then set your default album to `My Pixel 8 - Camera` to get a default view of the items you've used your camera for.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by deralex92** [comment]
Follow

**Comment by balvant813** [comment]
allows login [page default to /albums instead of /photos .... This will help in sending web link to guest logins to see albums and avoid teaching them how to switch from photos to album page.

**Comment by ghost** [comment]
This would be a very welcome feature.
Currently it is confusing for a guest to start immich!

**Comment by MikeSoftZP** [comment]
+1 here. 
missing the ability to specify start page/album/map

**Comment by HalloweenMagic** [comment]
Please, add this feature in app! :)
